__tests__/Snapshottest/__snapshots__/UIOutputComparisonTests.tsx.snap

```diff
@@ -112,7 +123,6 @@ exports[`LibraryContainer Test UI rendering of Library Item and child components
       "RefreshLibraryViewRequestName": "refreshLibraryView",
       "SearchTextUpdatedEventName": "searchTextUpdated",
       "SectionIconClickedEventName": "sectionIconClicked",
-      "FilterCategoryEventName": "filterCategoryChange",
```

Comment:
@ramramps I noticed this was also a recent addition in #116.  Tests also fail for me if these are not in this order, appears to be alphabetical

Reply:
No it needs to be present but the order in which the objects are listed appears to influence the test results.